src/oic/utils/http_util.py

```diff
@@ -437,12 +438,18 @@ def __init__(self, srv, ttl=5):
         self.pad_chr = " "
 
     def init_srv(self, srv):
-        if srv:
-            self.srv = srv
-
-            for param in ["seed", "iv"]:
-                if not getattr(srv, param, None):
-                    setattr(srv, param, rndstr().encode("utf-8"))
+        if not srv:
+            return
```

Comment:
Because the original code did not either and I wasn't sure if this was acceptable. I'll patch that up. Please, @rohe, can you confirm this?